issue_template.md

```diff
@@ -1,29 +1,41 @@
-### Summary
-
--   What does this package do? (explain in 50 words or less):
+Submitting Author: Name (@github_handle)  
+Repository:   
+Version submitted:   
+Editor: TBD  
+Reviewer 1: TBD  
+Reviewer 2: TBD  
+Archive: TBD  
```

Comment:
will it be clear to submitters what __Archive__ means?

Reply:
Yes. and we can explain it in the docs. Zenodo archive makes it too prescriptive. Again in the docs we can explain in more details what it means to archive. Also in the initial template all of this is marked as TBD which means that no one will have to worry about filling this out.